styleguide/components/21-atoms/skiplink/_skiplink.scss

```diff
@@ -3,7 +3,7 @@ a.skiplink {
   top: -100%;
   left: 0;
   padding: .5rem 2rem;
-  background-color: color('white');
+  background: color('white');
```

Comment:
What's wrong with background-color?

Reply:
Nothing in essence, but the skiplink inherits the background from the link component including the new gradient. Since this is not desired, we override the entire background instead of just the background-color.